packages/react-components/eslint-plugin-react-components/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from 'node:fs';
+import * as path from 'node:path';
```

Comment:
lets switch to named exports. anything `*` is potential tree shaking issue

Reply:
what if we'll use something like this `import { name, version } from '../package.json'` instead, wdyt?